File: C++/054. AprovadoReprovado/AprovadoReprovado.cpp
#include <iostream>
int main()
{
    float N;
    int F;
    std::cin >> N >> F;
    if (N >= 6.0 && F <= 30) {
        std::cout << "Aprovado!" << std::endl;
        return 0;
    }
    else
    {
        std::cout << "Reprovado!" << std::endl;
        return 0;
    }
}

File: Java/054. AprovadoReprovado/AprovadoReprovado.java
import java.util.Scanner;
public class AprovadoReprovado
{
    public static void main(String[] args) 
    {
        Scanner scan = new Scanner(System.in);
        double media = scan.nextDouble();
        int faltas = scan.nextInt();
        if (media >= 6.0 && faltas <= 30)
        {
            System.out.println("Aprovado!");
        }
        else
        {
            System.out.println("Reprovado!");
        }
        scan.close();   
    }
}